packages/components/buttons/LargeBoxButton.tsx

```diff
@@ -51,7 +48,7 @@ export const LaunchpadButton: React.FC<LaunchpadButtonProps> = ({
             <SVG source={ChevronRightSvg} />
           </View>
         </View>
-      </LegacyTertiaryBox>
+      </Box>
```

Comment:
Yes, we should remove the legacy deprecated stuff when it's on our way, thx for that